composer.json

```diff
@@ -10,6 +10,7 @@
         "php": "^7.2",
         "ext-dom": "*",
         "ext-libxml": "*",
+        "erebot/dom": "^0.1.0",
```

Comment:
who knew that you could program in XML https://github.com/Erebot/DOM/blob/master/data/iso_schematron_skeleton_for_xslt1.xsl

Reply:
are the two licenses:
- https://github.com/Erebot/DOM/blob/master/LICENSE (GPL for PHP code)
- https://github.com/Erebot/DOM/blob/master/LICENSE.skeleton ([Zlib](https://en.wikipedia.org/wiki/Zlib_License) I think, for the XML)
compatible with Libero?

Reply:
This was more a POC rather than a suggestion to start relying on that library, since it doesn't look very mature.

Reply:
The XSLT itself looks to be available in 
https://github.com/Schematron/schematron under MIT.

Reply:
To avoid having to try and distribute a GPL dependency we'll have to create our own library.

Reply:
Creating https://github.com/libero/xml-validator.

Reply:
And implemented in https://github.com/libero/xml-validator/pull/6.